src/core/client/admin/routes/Moderate/Queue/ForReviewQueueRoute/ForReviewQueueRow.tsx

```diff
@@ -0,0 +1,159 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+
+import CommentParser from "coral-admin/components/Comment/CommentParser";
+import { getLocationOrigin } from "coral-framework/utils";
+import { CheckBox, Flex, TableCell, TableRow } from "coral-ui/components/v2";
+import { TimestampFormatter } from "coral-ui/components/v2/Timestamp";
+import { ComponentLink } from "coral-ui/components/v3";
+
+import { COMMENT_FLAG_REASON } from "coral-admin/__generated__/ForReviewQueueRoute_query.graphql";
+
+import styles from "./ForReviewQueueRow.css";
+
+interface ForReviewRevision {
+  readonly body: string | null;
+}
+
+interface ForReviewComment {
+  id: string;
+  revision: ForReviewRevision;
+}
+
+interface Props {
+  id: string;
+  comment: ForReviewComment | null;
+  username: string | null;
+  reason: COMMENT_FLAG_REASON | null;
+  additionalDetails: string | null;
+  createdAt: string;
+  reviewed: boolean;
+  onReview: (id: string, enabled: boolean) => void;
+}
+
+const getRevision = (comment: ForReviewComment | null) => {
```

Comment:
`getRevision` is inaccurate, as we are only returning the `body`. So `getRevisionBody` would be more accurate, but I think it should be just named `getCommentBody` instead.

Reply:
Refactored this away during container refactor you requested elsewhere